kano_init_flow/stages/bugs/main.py

```diff
@@ -110,4 +110,5 @@ def _bug_zapped(self, scene, wid):
             scene.remove_widget('speech-bubble')
             self._place_judoka_into_scene(scene, True)
 
-            scene.schedule(3, self._ctl.next_stage)
+            # the gif is so slow to load, no need to a timeout
+            scene.schedule(0, self._ctl.next_stage)
```

Comment:
It would be better to just call `self._ctl.next_stage()` rather then scheduling it with no delay.